uber/site_sections/preregistration.py

```diff
@@ -807,15 +806,17 @@ def abandon_badge(self, session, id):
                                        failure_message)
                 elif response:
                     amount_refunded += response.amount
+                    for receipt_item in [item for item in attendee.receipt_items if item.txn_type == c.PAYMENT]:
```

Comment:
When you originally implemented the ``txn_type`` field, it made sense to me, but for some reason my brain now always expands it to "taxation type".